sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -79,7 +79,7 @@ export const InstitutionRoutesConst = {
   STUDENT_APPLICATION_DETAILS: Symbol(),
   STUDENT_RESTRICTIONS: Symbol(),
   STUDENT_FILE_UPLOADS: Symbol(),
-  STUDENT_OVERAWARDS: Symbol(),
+  STUDENT_BALANCES: Symbol(),
```

Comment:
In my APi i have used Student loan balances, just to be more understandable. Not a blocker on the naming, but just a suggestion.

Reply:
This route is corresponding to the tab which was previously called overawards and now called balances.

![image](https://github.com/bcgov/SIMS/assets/54600590/5b7c413c-c8c9-4d88-87ad-faee43408dbd)